src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.WriteTests.cs

```diff
@@ -5,31 +5,32 @@
 using System.IO;
 using System.Text.Json.Serialization.Tests.Schemas.OrderPayload;
 using System.Threading.Tasks;
+using System.Text.Json.Serialization.Metadata;
 using Xunit;
 
 namespace System.Text.Json.Serialization.Tests
 {
     public partial class StreamTests
     {
         [Fact]
-        public static async Task WriteNullArgumentFail()
+        public async Task WriteNullArgumentFail()
```

Comment:
Unrelated to the change itself, but these test weren't being run because they were marked as static.